src/backend/base/langflow/components/nvidia/nemo_evaluator.py

```diff
@@ -293,38 +372,38 @@ async def evaluate(self) -> dict:
         if evaluation_type == "LM Evaluation Harness":
             data = self._generate_lm_evaluation_body()
         elif evaluation_type == "Custom Evaluation":
-            data = self._generate_custom_evaluation_body()
+            data = await self._generate_custom_evaluation_body()
```

Comment:
why is this being made async? I don't understand changes being made to sync vs. async in this PR

Reply:
Custom evaluation runs using a dataset in datastore. Similar to customizer component, I have made this to accept LF `Data` object as input which internally writes into datastore.